interface-definitions/protocols_openfabric.xml.in

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="protocols">
+    <children>
+      <node name="openfabric" owner="${vyos_conf_scripts_dir}/protocols_openfabric.py">
+        <properties>
+          <help>OpenFabric protocol</help>
+          <priority>680</priority>
+        </properties>
+        <children>
+          #include <include/openfabric/interface.xml.i>
+          <leafNode name="net">
```

Comment:
Same as in IS-IS, so a good candidate to make this a building block as it contains a complex regex.